MathLib/KelvinVector.h

```diff
@@ -235,6 +235,13 @@ KelvinVectorType<DisplacementDim> symmetricTensorToKelvinVector(
             1));
 }
 
+/// Conversion of a symmetric tensor, which is stored in a column of Eigen
+/// matrix with a row size of 4 or 6, to a full tensor in a 3x3 matrix
+template <int KelvinVectorSize>
+Eigen::Matrix<double, 3, 3> symmetricTensorToFullTensor(
```

Comment:
This is wrong place for this function which has nothing to do with Kelvin-mapped vectors.

Reply:
Moved to MathTools.h